ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -449,6 +456,17 @@ export default defineComponent({
 
       // If already in review mode, file the transfer
       if (localState.isReviewMode) {
+        // Verify no lien exists prior to submitting filing
+        const regSum = !localState.isJestRunning
```

Comment:
Could we check for Jest at the same time we check for isReviewMode, line 458?

Reply:
We could* although that would prevent all the interactions within this block, inside our unit tests. 
Example being, triggering the error state for required fields which in turn displays validation messaging etc.

Reply:
Sounds good, thanks for clarifying. Resolving.

Reply:
No problem at all, a very valid question!